lib/Elastica/Suggest/AbstractSuggest.php

```diff
@@ -16,9 +17,9 @@
     protected $_name;
 
     /**
-     * @var string the text for this suggestion
+     * @var string the prefix for this suggestion
      */
-    protected $_text;
+    protected $_prefix;
```

Comment:
Plan SGTM. @Zyqsempai Any objections? If not, will merge #1213 and for 2 and 3 we can use this or future PR's.